shared/key-codes.js

```diff
@@ -0,0 +1,16 @@
+(function (window) {
```

Comment:
I would be better if we name this module 'constants' or something like that, where we could store all those values. So you could use afterwards Constants.KeyCodes.ENTER and Constants.Color.RED from the same import.